doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,65 +3,46 @@
-
-Steps to reproduce:
-
-. Use the installer to install {PRODUCT} on {JWS}.
-. When prompted, click *Configure Advanced Properties* -> *Configure Database Settings*.
-. Configure your settings and add a user name and password.
-. Try running the server.
-
-Expected result: {PRODUCT} starts.
-
-Actual result: {PRODUCT} does not start.
-
-Workaround: Modify the resource in the `context.xml` file as described in the "Configuring JDBC Web Server data sources" section of {URL_INSTALLING_ON_JWS}[_{INSTALLING_ON_JWS}_].
-
-//approved
-.On {EAP}, the {PRODUCT} installer creates an incorrect password vault for the created datasource [https://issues.jboss.org/browse/RHPAM-2407[RHPAM-2407]]
+== {CENTRAL}
 
-Issue: If you use the installer to install {PRODUCT} on {EAP} and you configure database settings, the `datasource` element in the `standalone` files is wrong and connection to the database fails.
+.When you add a deployment unit and manually enter GAV values, the runtime strategy is not set to the configured default value [https://issues.redhat.com/browse/RHPAM-2623[RHPAM-2623]]
 
-NOTE: This issue is resolved with {PRODUCT} patch release 7.5.1.
+Issue: In {CENTRAL}, you can configure a default runtime strategy. However, when you add a new deployment unit and enter the group, artifact, and version values manually, the runtime strategy is not set to this configured default.
 
 Steps to reproduce:
 
-. Use the installer to install {KIE_SERVER} only onto {EAP}.
-. When prompted, click *Configure Advanced Properties* -> *Configure Database Settings*.
-. Open the `standalone.xml` or `standalone-full.xml` file to verify user credentials.
-
-Expected result: The datasource element in the `standalone` files should look similar to the following, where `<NEWDATASOURCE>` is the name of your database:
-[source]
-----
-<password>${VAULT::datasource.<NEWDATASOURCE>::password::1}</password>
-----
-
-Actual Result: The datasource element in the `standalone` files looks similar to the following:
-
-[source]
-----
-<password>:datasource.<NEWDATASOURCE>::password::1</password>
+. Import the *Evaluation_Process* example in {CENTRAL}.
+. In the main menu, select *Settings* -> *Deployments* -> *General Settings*.
+. Change *Runtime Strategy* to `Per Process Instance`.
+. Click *Build&Install*.
+. In the main menu, select *Execution Servers* -> *Add Deployment Unit*.
+. Enter values into the following fields:
+** *Name*: `evaluation:evaluation:1.0.0-SNAPSHOT`
+** *Group Name*: `evaluation`
+** *Artifact Id*: `evaluation`
+** *Version*: `1.0.0-SNAPSHOT`
+. Click *Next* (the result is not changed if you click *Finish* instead, creating the unit immediately).
+. In the window, note that the *Runtime Strategy* is `Singleton`.
+. Click *Finish*.
+. Start the deployment.
+. When the deployment is complete, click the deployment link (for example, `http://localhost:8080/kie-server/services/rest/server/containers/evaluation:evaluation:1.0.0-SNAPSHOT`)
```

Comment:
Remove () and add comma. Also, I would change the verb, and make the link not clickable,, and add a period:

After the deployment has completed, click the deployment link, for example `\http://localhost:8080/kie-server/services/rest/server/containers/evaluation:evaluation:1.0.0-SNAPSHOT`.